src/action.js

```diff
@@ -8,6 +8,14 @@ const { auth: { retrieveToken }, secrets: { getSecrets } } = require('./index');
 const AUTH_METHODS = ['approle', 'token', 'github', 'jwt', 'kubernetes'];
 const ENCODING_TYPES = ['base64', 'hex', 'utf8'];
 
+function addMask(value) {
```

Comment:
I think moving this to its own function is a good idea, however it seems to cause some issues with the acceptance tests

```
~$ npm run test

> vault-action@0.1.0 test
> jest

 PASS  src/auth.test.js
 FAIL  src/action.test.js
  ● exportSecrets › single-line secret gets masked

    expect(jest.fn()).toBeCalledTimes(expected)

    Expected number of calls: 1
    Received number of calls: 2

      334 |         await exportSecrets();
      335 |
    > 336 |         expect(command.issue).toBeCalledTimes(1);
          |                               ^
      337 |
      338 |         expect(command.issue).toBeCalledWith('add-mask', 'secret');
      339 |         expect(core.setOutput).toBeCalledWith('key', 'secret');

      at Object.toBeCalledTimes (src/action.test.js:336:31)

  ● exportSecrets › multi-line secret gets masked for each line

    expect(jest.fn()).toBeCalledTimes(expected)

    Expected number of calls: 2
    Received number of calls: 4

      354 |         await exportSecrets();
      355 |
    > 356 |         expect(command.issue).toBeCalledTimes(2); // 1 for each non-empty line.
          |                               ^
      357 |
      358 |         expect(command.issue).toBeCalledWith('add-mask', 'a multi-line string');
      359 |         expect(command.issue).toBeCalledWith('add-mask', 'with blank lines');

      at Object.toBeCalledTimes (src/action.test.js:356:31)

::add-mask::value
::add-mask::value
 PASS  src/retries.test.js

Test Suites: 1 failed, 2 passed, 3 total
Tests:       2 failed, 27 passed, 29 total
Snapshots:   0 total
Time:        2.532 s, estimated 3 s
Ran all test suites.
::add-mask::value
::add-mask::value
```